functions/migrate.go

```diff
@@ -25,10 +25,7 @@ func Migrate() {
 	if err != nil {
 		slog.Warn("set hostname", "error", err)
 	}
-	config_dir := config.GetNetclientPath()
-	if !ncutils.IsWindows() {
-		config_dir += "config"
-	}
+	config_dir := config.GetNetclientPath() + "config/"
```

Comment:
won't the forward slash cause an issue on windows?

Reply:
on windows the config dir is not used by netclient in normal operations.  During migration, the existence of the directory triggers the migration process.  os.Stat() does not seem to mind the slash